app/Http/Middleware/ViewBook.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Session\Store;
+use Session;
+
+class ViewBook
+{
+    protected $session;
+
+    public function __construct(Store $session)
+    {
+        $this->session = $session;
+    }
+
+    public function handle($request, Closure $next)
+    {
+        $book = $this->getViewedBooks();
+
+        if (!is_null($book))
+        {
+            $book = $this->cleanExpiredViews($book);
+            $this->storeBooks($book);
+        }
+
+        return $next($request);
+    }
+
+    private function getViewedBooks()
+    {
+        return $this->session->get('viewed_book', null);
+    }
+
+    private function cleanExpiredViews($book)
+    {
+        $time = time();
+        $throttleTime = 3600;
+
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnNewLine</br>**Severity**: error</br>**File**: app/Http/Middleware/ViewBook.php</br>**Line**: 42</br>Opening brace should be on the same line as the declaration